.sonarcloud.properties

```diff
@@ -0,0 +1,10 @@
+# For Auto-scan/Pull-Request Scan.
```

Comment:
I'm not sure we want to do automatic scans. I set this up in Nunavut to try to get PR scanning to work but it didn't. See https://sonarcloud.io/documentation/analysis/pull-request/ for how to enable PR scanning correctly.

Reply:
OK

Reply:
As discussed in slack, let's just yank this file for this PR.